src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/storm/bolt/isl/IslHandler.java

```diff
@@ -193,12 +199,9 @@ public void auxiliaryPollModeUpdateRequest(Endpoint endpoint, boolean enableAuxi
     }
 
     @Override
-    public void exhaustedPollModeUpdateRequest(Endpoint endpoint, boolean enableExhaustedPollMode) {
-        WatchListCommand command = new WatchListExhaustedPollModeUpdateCommand(endpoint, enableExhaustedPollMode);
-        // We emit without the anchor tuple because here we are generating a new event to change the mode.
-        // Also, if a cycle appears in the future by the anchor tuple, it will be quite difficult to find it,
-        // and we remove the possibility of this cycle appearing initially.
-        emit(STREAM_POLL_ID, makePollTuple(command));
+    public void islRemovedNotification(Endpoint srcEndpoint, IslReference reference) {
+        emit(STREAM_UNIISL_ID, getCurrentTuple(),
```

Comment:
loop... have been created.

Reply:
Good catch. Thank you. Loop has been removed.